lib/index.js

```diff
@@ -66,6 +65,7 @@ PG.prototype.connect = function(config, callback) {
   config.idleTimeoutMillis = config.idleTimeoutMillis || config.poolIdleTimeout || defaults.poolIdleTimeout;
   config.log = config.log || config.poolLog || defaults.poolLog;
 
+  var poolName = JSON.stringify(config || {});
```

Comment:
Do you want me to upgrade this PR including that dependency? Or should we use another PR to do that? I prefer to use another PR as it's a separate issue, but I'm fine managing that one 😄 .